src/main/java/com/textmessenger/HealthCheckController.java

```diff
@@ -1,12 +1,13 @@
 package com.textmessenger;
 
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.springframework.web.bind.annotation.RequestMapping'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=145&issues=AWWKTQcw7khxiKmGRw4a&open=AWWKTQcw7khxiKmGRw4a)

[ISSUE_KEY]: AWWKTQcw7khxiKmGRw4a